modules/payloads/singles/php/meterpreter_reverse_tcp.rb

```diff
@@ -39,6 +39,17 @@ def generate
     session_guid = '\x00' * 16
     met = met.sub(%q|"SESSION_GUID", ""|, %Q|"SESSION_GUID", "#{session_guid}"|)
 
+    ds = datastore
+    ds ||= framework.datastore
+
+    if (ds['MeterpreterDebugBuild'])
+      payload.gsub!("define(\"DEBUGGING\", false);", "define(\"DEBUGGING\", true);")
```

Comment:
Just confirming, should this be `met` ?

Reply:
Thoughts on replacing the hard coded `true` to be `ds['MeterpreterDebugBuild'].to_s', or similar, to ensure the boolean value is set

Reply:
Confirmed it crashes for me:
```
msf6 payload(php/meterpreter_reverse_tcp) > generate -o shell.sh -f raw MeterpreterTryToFork=false PythonMeterpreterDebug=true MeterpreterDebugBuild=true

[-] Payload generation failed: undefined method `gsub!' for nil:NilClass
```